tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/api/rest/v1/controllers/NewController.java

```diff
@@ -33,11 +36,35 @@ public class NewController {
 
```

Comment:
Is there any reason to call this class "NewController"?

Would be possible rename by the domain like `PublicProcessController` ? 

What do you think?